### DISCUSSION TITLE
Remeshing support

### DISCUSSION THREAD

**Discussion by l-matSci** [neperfepx/neper#663] [Q&A]
Hello,

I am quite new to FEPX/NEPER Project. First of all, i want to thank the developers for providing these tools to the community!

I am interested in doing a simulation setup with large strains, which will require remeshing.

On the FEPX Q&A page, i read that remeshing is currently not supported in the current NEPER release, although the post is from December 2021.
(https://github.com/neperfepx/FEPX/discussions/63), although the post is from December 2021.

Are there plans to reintroduce the remeshing, inlcuding nodal variable transfer? 
Thank you.

**Comment by rquey** [comment]
The remeshing capabilities are still present in Neper (load a mesh as input in `-M` and use `-transport` to transfer variables), but it does not interface well with (the new version of) FEPX yet. It is not top-priority at the moment, but it should be back in a relatively near future.

**Reply by l-matSci** [reply]
Thank you, Romain.

What is the issue with the interface between NEPER/FEPX? is it about restartig or the actual mesh/tesselation files proveded to the Solver?

**Reply by rquey** [reply]
The current version of FEPX can read the transported orientations, but nothing more (not the transported slip strength parameters, etc.) - this isn't something we have allowed yet.
